main.js

```diff
@@ -0,0 +1,321 @@
+const matrix = [
```

Comment:
it would be better to have object with key value, in current realisation it is not clear what set of codes is used for

Reply:
Not decided. Mistakenly clicked.